Decompiler/PscCoder.cpp

```diff
@@ -56,32 +55,6 @@ Decompiler::PscCoder &Decompiler::PscCoder::outputAsmComment(bool commentAsm)
     return *this;
 }
 
-/**
- * @brief Write the content of the PEX header as a block comment.
- * @param pex Binary to decompile.
- */
-void Decompiler::PscCoder::writeHeader(const Pex::Binary &pex)
```

Comment:
There was a specific question in the discord about why this was removed in one of the prior commits, and it probably makes sense to add it back, though the version info will need to be updated for the new champollion version number.